### PR TITLE
Translations for Portuguese Brazil

### PR DESCRIPTION
**Pull request by juliozuppa**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Related issue(s):
1.
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by AMBudnik** [comment]
We still need to wait for another 3 confirmations from users.

**Comment by AMBudnik** [comment]
Thank you for your input @FreitasMurillo

**Comment by AMBudnik** [comment]
Thank you for checking the PR @ElHoro @allyssonm

**Comment by AMBudnik** [comment]
Thank you very much for your confirmations. I am checking the PR status to `ready for review`

File: src/i18n/languages/pt-BR.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Júlio C. Zuppa
+ * Last updated: Jan 12, 2018
+ *
+ * Description: Definition file for brazilian portuguese - Brazil language-country.
```
Comment (wszymanski) [inline src/i18n/languages/pt-BR.js]: Please change from: `Description: Definition file for brazilian portuguese - Brazil language-country.` to `Description: Definition file for Portuguese - Brazil language-country.`

**Comment by adrianszymanski89** [comment]
Hi @juliozuppa. We just announced an introduction of a new functionality called `Copy with headers`. Could you help us to translate three short lines for the context menu?

Here you can find more information about it: https://github.com/handsontable/handsontable/issues/10112

**Pull request merged**